include/blmc_robots/real_finger.hpp

```diff
@@ -373,4 +388,188 @@ robot_interfaces::finger::BackendPtr create_real_finger_backend(
     return backend;
 }
 
+class RandomFingerDriver : public robot_interfaces::RobotDriver<
+                              robot_interfaces::finger::Action,
+                              robot_interfaces::finger::Observation>
+{
+
+public:
+    typedef robot_interfaces::finger::Action Action;
+    typedef robot_interfaces::finger::Observation Observation;
+    typedef robot_interfaces::finger::Vector Vector;
+    typedef robot_interfaces::finger::Status Status;
+
+    //default constructor, error: implicitly deleitng
+    RandomFingerDriver() : RobotDriver(0.003, 0.005) {}
+
+    int i = 0;
```

Comment:
Please use a more descriptive name.  Also, according to our style guide, class variables should be suffixed with `_` (e.g. `int foobar_`).

For better overview, the class variables should be listed before the methods, i.e. please move it before the constructor .